mu-plugins/global-fonts/index.php

```diff
@@ -257,9 +261,6 @@ function get_font_url( $font, $subset ) {
 
 	$filepath = $font_folder . $font_file_name . $lower_subset . '.woff2';
 	if ( ! file_exists( __DIR__ . '/' . $filepath ) ) {
-		if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
-			trigger_error( sprintf( 'Requested font file %s does not exist.', esc_html( $filepath ) ), E_USER_WARNING );
-		}
```

Comment:
This is not a helpful error message. Ideally it would be flagged to the translators, because it seems to trigger most often when they've translated the subset into their language, but instead we can silently ignore it. The only effect of ignoring it is that the font is not _pre_loaded, but it is still loaded.